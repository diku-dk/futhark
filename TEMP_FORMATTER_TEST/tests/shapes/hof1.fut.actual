509aa74efcac16fbe23b6b9d3c47285c
