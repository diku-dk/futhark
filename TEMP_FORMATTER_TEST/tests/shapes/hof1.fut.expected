f22ebcc36265f8a535c87b8da4dddf8f
